framework/PostProcessors/LimitSurfaceIntegral.py

```diff
@@ -216,25 +223,25 @@ def inputToInternal(self, currentInput):
      @ In, currentInput, object, an object that needs to be converted
      @ Out, None
     """
-    for item in currentInput:
-      if item.type == 'PointSet':
-        self.matrixDict = {}
-        if not set(item.getParaKeys('inputs')) == set(self.variableDist.keys()):
-          self.raiseAnError(IOError, 'The variables inputted and the features in the input PointSet ' + item.name + 'do not match!!!')
-        if self.target == None:
-          self.target = item.getParaKeys('outputs')[-1]
-        if self.target not in item.getParaKeys('outputs'):
-          self.raiseAnError(IOError, 'The target ' + self.target + 'is not present among the outputs of the PointSet ' + item.name)
-        # construct matrix
-        for  varName in self.variableDist.keys():
-          self.matrixDict[varName] = item.getParam('input', varName)
-        outputarr = item.getParam('output', self.target)
-        if len(set(outputarr)) != 2:
-          self.raiseAnError(IOError, 'The target ' + self.target + ' needs to be a classifier output (-1 +1 or 0 +1)!')
-        outputarr[outputarr == -1] = 0.0
-        self.matrixDict[self.target] = outputarr
-      else:
-        self.raiseAnError(IOError, 'Only PointSet is accepted as input!!!!')
+    item = currentInput[0]
```

Comment:
I think we should check also the number of inputs.

Reply:
addressed